extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +122,32 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  handleYoutubeMessages_(event) {
+    if (event.origin != 'https://www.youtube.com' ||
+        event.source != this.iframe_.contentWindow) {
+      return;
+    }
+    const data = JSON.parse(event.data);
+    if (data.event == 'onReady') {
+      this.playerReadyResolver_(this.iframe_);
+    } else if (data.event == 'infoDelivery') {
+      if (data.info && data.info.playerState !== undefined) {
```

Comment:
can probably get rid of this `if` and just inline the other conditions with the `else if` right?